apps/verticalhome/js/pin_apps_default.json

```diff
@@ -0,0 +1,33 @@
+{"pin_apps_default":
+[
+  {"entry_point": "dialer",
+    "name": "Phone",
+    "manifestURL": "app://communications.gaiamobile.org/manifest.webapp",
+    "icon":   "app://communications.gaiamobile.org/dialer/style/icons/dialer_84.png",
+    "index": 0
```

Comment:
Seems like this index is unnecessary since these are array elements that have an index implicitly.